mmv1/third_party/terraform/resources/resource_google_project.go

```diff
@@ -179,7 +185,14 @@ func resourceGoogleProjectCreate(d *schema.ResourceData, meta interface{}) error
 	// a network and deleting it in the background.
 	if !d.Get("auto_create_network").(bool) {
 		// The compute API has to be enabled before we can delete a network.
-		if err = enableServiceUsageProjectServices([]string{"compute.googleapis.com"}, project.ProjectId, userAgent, config, d.Timeout(schema.TimeoutCreate)); err != nil {
+
+		billingProject := project.ProjectId
```

Comment:
Can we split the `billing_project` support and retrying into separate PRs? Ack that they're near the same thing, but they'll both be conceptually simpler. Also, retrying very likely needs to touch the _operation_ code, not the mutate request code see my other comment.

Reply:
done, I've made this pr exclusive to billing_project